src/CMakeLists.txt

```diff
@@ -182,7 +181,7 @@ endif(COMPILE_SERVER)
 foreach(exec ${EXECUTABLES})
   target_link_libraries(${exec} marian ${EXT_LIBS} ${EXT_LIBS} ${CMAKE_THREAD_LIBS_INIT})
   if(CUDA_FOUND)
-    target_link_libraries(${exec} marian marian_cuda ${EXT_LIBS} ${CMAKE_THREAD_LIBS_INIT})
+    target_link_libraries(${exec} marian marian_cuda nccl ${EXT_LIBS} ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
Note to self: why is CMake not screaming at me that target nccl or libnccl does not exist when I am building with -DUSE_NCCL=off ?